models/volunteer.py

```diff
@@ -0,0 +1,95 @@
+import datetime
+from models import camp
+
+
+class Volunteer:
+    """
+    A class used to represent a volunteer.
+    """
+
+    def __init__(self, firstname: str,
+                 lastname: str,
+                 phone: str,
+                 volunteercamp: list,
+                 availability = True):
+        """
+           Initialize a new volunteer.
+           :param firstname: the firstname of the volunteer
+           :param lastname: the lastname of the volunteer
+           :param phone: the phone number of the volunteer
```

Comment:
Would be worth clarifying here that only numbers with international code are accepted i.e. this is not just a numerical string

Reply:
Thanks for the comment! will update it at the new branch "volunteerAccount